src/layouts/Header/ControlBar/ControlBar.jsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+function ControlBar() {
+  return <div></div>;
+}
+
+export default ControlBar;
```

Comment:
Where is this component on your design? What does this component do? Maybe you need to rename it.

Reply:
Removed this component. He is redundant here